usedapp-rootstock/README.md

```diff
@@ -0,0 +1,28 @@
+# Build on Rootstock with useDApp
+
+This demo repo shows how to build a DApp on Rootstock using [useDApp](https://usedapp.io/) React library
```

Comment:
```suggestion
This demo repo shows how to build a DApp on Rootstock using [useDApp](https://usedapp.io/) React library.

Note that instead of specifying the configuration manually within the code,
instead you can do the following, to obtain the configuration for Rootstock
directly from useDApp package:

`import { RootstockMainnet, RootstockTestnet } from '@usedapp/core';`

```